src/assets/data/recipes.json

```diff
@@ -0,0 +1,728 @@
+[
+  {
+    "recipe_id": 1,
+    "recipe_type": "Dessert",
+    "recipe_name": "Chocolate Cake",
```

Comment:
please add a "slug" entry in order to provide an url segment for recipe detail e.g: "chocolate-cake"